core/util.py

```diff
@@ -9,13 +11,16 @@
 POSTCODE_REGEX = r"[A-Z]{1,2}[0-9][A-Z0-9]? ?[0-9][A-Z]{2}"
 
 
-def extract_postcodes(s: str) -> list[str]:
+def extract_postcodes(s: Union[str, float]) -> List[str]:
```

Comment:
Minor point but Python 3.11 introduced a new (and preferred notation for typing) - `Union[str, float]` can now be `str | float`

and also we don't need to use `List` any more - can just do `list[str]`

Reply:
seems really insignificant in the scheme of today but the change you made is slightly wrong.. `Union[str | float]` should be just `str | float`